config/routes.rb

```diff
@@ -1,56 +1,7 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  resources :reports, only: [] do
+    get :call_volumes, on: :collection, as: :call_volumes
```

Comment:
This feels like a whole load of gymnastics to fit in with the `resources` world view. Wouldn't this be simpler if we just used `get 'reports/call_volumes', to: 'reports#call_volumes'` for now?